README.md

```diff
@@ -79,26 +77,208 @@ add a link to the PR in the header of the RFC.
 are much more likely to make progress than those that don't receive any
 comments.
 * Eventually, the [core teams] will decide whether the RFC is a candidate
-for inclusion in Ember.
-* RFCs that are candidates for inclusion in Ember will enter a "final comment
-period" lasting 7 days. The beginning of this period will be signaled with a
+for advancement to the next stage in the RFC process.
+* Upon advancement, the RFC's pull request will be merged.
+* When the RFC is ready to continue advancing to the next stage, open a new PR
+advancing the RFC by updating the stage, and making any additional changes that
+are necessary.
+* The RFC continues through the process until it either reaches the final stage,
+or is closed or discontinued.
+
+### Final comment periods
+
+For certain stage advancements, a _final comment period_ (FCP) is required. This 
+is a period lasting 7 days. The beginning of this period will be signaled with a
 comment and tag on the RFC's pull request. Furthermore,
 [Ember's official Twitter account](https://twitter.com/emberjs) will post a
 tweet about the RFC to attract the community's attention.
-* An RFC can be modified based upon feedback from the [core teams] and community.
-Significant modifications may trigger a new final comment period.
-* An RFC may be rejected by the [core teams] after public discussion has settled
-and comments have been made summarizing the rationale for rejection. The RFC 
-will enter a "final comment period to close" lasting 7 days. At the end of the 
-"FCP to close" period, the PR will be closed.
-* An RFC may also be closed by the core teams if it is superseded by a merged
-RFC. In this case, a link to the new RFC should be added in a comment.
-* An RFC author may withdraw their own RFC by closing it themselves.
-* An RFC may be accepted at the close of its final comment period. A [core team]
-member will merge the RFC's associated pull request, at which point the RFC will
-become 'active'.
-
-### Relevant Teams 
+
+An RFC can be modified based upon feedback from the [core teams] and community
+during the final comment period. Significant modifications may trigger a new 
+final comment period.
+
+At the end of the FCP, the RFC advances to the next stage in the process.
+
+### FCP to close
+
+An RFC may be closed or discontinued by the [core teams] after public discussion 
+has settled and comments have been made summarizing the rationale for closing. 
+The RFC will enter a "final comment period to close" lasting 7 days. At the end 
+of the "FCP to close" period, the PR will be closed.
+
+An RFC author may withdraw their own RFC by closing it themselves.
+
+## The RFC Stages
```

Comment:
Just for ease of community use/understanding, what do you think about moving the table above some of the other topics in the README?